public/models/game/game.js

```diff
@@ -59,6 +59,18 @@ var Game = Map.extend({
 				newVal.__listSet = {gameId: this.attr("id")};
 				return newVal;
 			}
+		},
+		videoUrl: {
+			serialize: true,
+			get: function (setVal) {
```

Comment:
I don't fully understand the question. I test the value of the property after the `set` using `.attr('videoUrl`)`. How would the existence of a setter change how I test what's sent to the server?